mlir/lib/Dialect/Math/Transforms/CanonicalizeF32Promotion.cpp

```diff
@@ -0,0 +1,73 @@
+//===- CanonicalizeF32Promotion.cpp - Remove redundant extf/truncf pairs
+//----------===//
```

Comment:
remove "-------" to move this line to above line, for formatting?

Reply:
thanks for the comment. removed "-------"